app/templates/components/edit-reminder.hbs

```diff
@@ -4,3 +4,4 @@
 {{#link-to "reminders.reminder" model.id}}
   <button class="spec-save-button" {{action "save"}}>save</button>
 {{/link-to}}
+<button class="spec-reset-button" {{action "reset"}}>reset</button>
```

Comment:
Can we add additional functionality that either only enables this button or only shows it if the attributes are dirty?